quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,514 @@
+"""
+Core recipes for the Q-Chem
+"""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Literal
+
+import covalent as ct
+from ase.atoms import Atoms
+from monty.dev import requires
+
+from quacc.calculators.qchem import QChem
+from quacc.schemas.ase import summarize_opt_run, summarize_run
+from quacc.util.calc import run_ase_opt, run_calc
+from pymatgen.io.ase import AseAtomsAdaptor
+
+try:
+    from sella import Sella, IRC
+except ImportError:
+    Sella = None
+
+
+def true_charge_and_spin(atoms, charge, mult):
+    mol = AseAtomsAdaptor.get_molecule(atoms)
+    if charge is not None:
```

Comment:
Not a bad idea. Chuck that into `quacc.util.atoms` so we can use it in other molecular DFT calculators in some way.